controllers/users/index.js

```diff
@@ -112,9 +123,38 @@ const getCurrentUser = async (req, res) => {
   }
 };
 
+const avatarChange = async (req, res) => {
+  try {
+    avatarUpload(req, res, async function (err) {
+      if (err instanceof multer.MulterError) {
+        return res.status(400).json({ message: "Multer error" });
+      } else if (err) {
+        return res.status(500).json({ message: err.message });
+      }
+
+      if (!req.file) {
+        return res.status(400).json({ message: "No file uploaded" });
+      }
+
+      const userId = req.user._id;
+
+      const image = await jimp.read(req.file.path);
+      await image.resize(250, 250).writeAsync(req.file.path);
+
+      const avatarURL = `/avatars/${req.file.filename}`;
```

Comment:
Tworzysz url, ale nie przenosisz pliku do public/avatars. Musisz jeszcze to zrobić. Czyli kiedy robisz resize i write podaj już nową scieżkę do folderu docelowego. I sprawdź czy url który otrzymujesz w postmanie wyświetli ci obrazek, czyli jak otrzymałem takie coś: 

`  "avatarURL": "/avatars/avatar_65e1feb79e9e57d3dfcb0e89_1714462708932.png"` 

To kiedy wejdę na http://localhost:3000/avatars/avatar_65e1feb79e9e57d3dfcb0e89_1714462708932.png
to powinienem zobaczyć w przeglądarce zdjęcie.

Reply:
W razie pytań pisz na slaku :)